projects/drivers/util/util.c

```diff
@@ -158,3 +158,21 @@ uint64_t do_div(uint64_t* n,
 
 	return mod;
 }
+
+uint64_t div64_u64_rem(uint64_t dividend, uint64_t divisor, uint64_t *remainder)
+{
+	*remainder = dividend % divisor;
+	return dividend / divisor;
```

Comment:
For consistency with the rest of the file I would add a new line.

Reply:
... and for the same reason, please add a short comment before the implementation.